WalletWasabi/WabiSabi/Client/CoinJoin/Client/CoinJoinClient.cs

```diff
@@ -126,6 +126,17 @@ private async Task<RoundState> WaitForBlameRoundAsync(uint256 blameRoundId, Canc
 			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the round is not economic.");
 		}
 
+		if (roundState.CoinjoinState.Parameters.CoordinationFeeRate.Rate * 100 > CoinJoinConfiguration.MaxCoordinationFeeRate)
+		{
+			Logger.LogWarning("The coordinator is malicious and tried to trick the client into paying a higher coordination fee rate in the blame round.");
+			throw new InvalidOperationException($"Blame Round ({roundState.Id}): Abandoning: the coordination fee rate is too high.");
```

Comment:
Add more info in the exception such as: the round fee rate and the expected fee rate. The error message is not that the coordination fee rate is too high but that the coordinator seems malicious.